### PR TITLE
Fixing temp path

### PR DESCRIPTION
**Pull request by dmvieira**
Was with // for path and filename

Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [ ] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-XXX

### Description
- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: airflow/operators/bash_operator.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def execute(self, context):
                 f.write(bytes(bash_command, 'utf_8'))
                 f.flush()
                 fname = f.name
-                script_location = tmp_dir + "/" + fname
+                script_location = fname
```
Comment (artwr) [inline airflow/operators/bash_operator.py]: maybe use `os.path.join` instead? Having the name of the temporary folder in the log can be useful for debugging

**Review by artwr — CHANGES_REQUESTED** [review]
I would love to find a solution that keeps the name of the temp directory.

File: airflow/operators/bash_operator.py (hunk 1 above)
Comment (bolkedebruin) [reply airflow/operators/bash_operator.py]: f.name returns the full path to the temp file (e.g. https://docs.python.org/3/library/tempfile.html#deprecated-functions-and-variables)

The script_location variable is redundant here and should be removed

**Review by bolkedebruin — COMMENTED** [review]
Small nit, please make sure to use a Jira.

File: airflow/operators/bash_operator.py (hunk 1 above)
Comment (bolkedebruin) [reply airflow/operators/bash_operator.py]:
```

>>> with TemporaryDirectory(prefix='airflowtmp') as tmp_dir:
...   with NamedTemporaryFile(dir=tmp_dir, prefix="my_prefix") as f:
...     print f.name
...
/var/folders/5y/z0j_5sn53_j1rhh_zhlhr8x80000gn/T/airflowtmpuc98p4/my_prefixEDzoYK

```

**Comment by bolkedebruin** [comment]
Any update?

**Comment by dmvieira** [comment]
Did

**Comment by bolkedebruin** [comment]
Jira issue? Squash your commits please!

**Comment by ron819** [comment]
@dmvieira please rebase and open a Jira ticket. All code changes PR require a Jira ticket.
https://issues.apache.org/jira/projects/AIRFLOW/issues

**Comment by dmvieira** [comment]
We can live without it

**Pull request closed without merging**